pipdeptree.py

```diff
@@ -266,6 +268,33 @@ def jsonify_tree(tree, indent):
                       indent=indent)
 
 
+def dumps_graphviz_dot(tree):
+    """Convert dependency graph to dot code for layout with graphviz.
+
+    :param dict tree: dependency graph
+    :returns: dot representation of tree
+    :rtype: str
+
+    """
+    import networkx as nx
```

Comment:
I agree, thanks to @Dunedan for addressing this in https://github.com/naiquevin/pipdeptree/pull/60/commits/0cbbee23bfc8584de240b23e6be71f16722c05c9#diff-c9a0cc375d0f079d6251af1de64f535bR356.